src/main.ts

```diff
@@ -40,6 +41,7 @@ async function processIssues(
     owner: github.context.repo.owner,
     repo: github.context.repo.repo,
     state: 'open',
+    labels: args.onlyLabels,
```

Comment:
This can be comma-separated list of labels:
https://octokit.github.io/rest.js/#octokit-routes-issues

Reply:
Mention that in the description!